containers/daap-python-base/src/var/task/data_platform_paths.py

```diff
@@ -131,7 +130,28 @@ def extract_database_name_from_curated_path(string: str):
 
 
 def extract_timestamp_from_curated_path(string: str):
-    return search_string_for_regex(string, regex=EXTRACTION_TIMESTAMP_CURATED_REGEX)
+    return search_string_for_regex(string, regex=LOAD_TIMESTAMP_CURATED_REGEX)
+
+
+def get_latest_version(data_product_name: str):
+    """Get the latest version of a data product, else return v1.0"""
+    metadata_bucket = get_metadata_bucket()
+    metadata_versions = s3.list_objects_v2(
+        Bucket=metadata_bucket, Prefix=data_product_name
```

Comment:
would the prefix simply be `data_product_name`? Is the metadata bucket going to just have data product metadata or will contain table schema as well? Currently the path is `metadata/{data_product_name}` but appreciate we only have the one bucket

Reply:
tbh I think even if we have schema as well it could use the same structure

I was imagining something like this

- {data_product_name}/v1.0/metadata.json
- {data_product_name}/v1.0/{table}/schema.json

Reply:
and if it is just the data product name it needs to end with `/` otherwise it'll pick up objects that start with that data product name
```suggestion
        Bucket=metadata_bucket, Prefix=data_product_name + "/"